src/views/mfa-verify/CallAndSMSForm.js

```diff
@@ -33,7 +33,7 @@ define(['okta', 'vendor/lib/q', 'views/shared/TextBox'], function (Okta, Q, Text
     },
 
     initialize: function () {
-      var self = this;
+      var form = this;
```

Comment:
Usage of `self` is far from the definition.
It makes easier to understand what is `self`, is it a form, model or button.

```
self.clearErrors();
this.disable();
this.render();
// To send an OTP to the device, make the same request but use
// an empty passCode
this.model.set('answer', '');
this.model.save()
.then(_.bind(function () {
this.options.title = formSubmitted;
// render and focus on the passcode input field.
self.getInputs().first().render().focus();
return Q.delay(API_RATE_LIMIT);
```
Is it easy to understand what is `self` here?